src/components/AppButton.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import { StyleSheet, Text, TouchableOpacity } from "react-native";
+import PropTypes from "prop-types";
+
+import { Colors } from "../constants/styles";
```

Comment:
To powinno importować cały plik o nazwie Colors z folderu styles ? Importowanie nie jest case sensitive ? Bo istniejacy plik ma nazwę colors.js

Reply:
ale w index.js eksportujemy Colors